src/explore-education-statistics-frontend/src/components/CookieBanner.tsx

```diff
@@ -26,24 +26,55 @@ function CookieBanner({ wide }: Props) {
           'dfe-width-container--wide': wide,
         })}
       >
-        <p>
-          <span>GOV.UK uses cookies to make the site simpler.</span>{' '}
-          <ButtonText
-            onClick={() => {
-              setBannerSeenCookie(true);
-              if (typeof getCookie('disableGA') === 'undefined') {
-                setGADisabledCookie(false);
-              }
-            }}
-          >
-            Accept Cookies
-          </ButtonText>{' '}
-          or{' '}
-          <Link to="/cookies">
-            find out more about cookies and cookie settings
-          </Link>
-          .
-        </p>
+        <div
+          className="govuk-cookie-banner "
```

Comment:
The styling for this banner looks wrong. For a start, it's way too blue because of `styles.container`.

Additionally, the `govuk-cookie-banner` styles were only added in `govuk-frontend@3.11.0`:

https://github.com/alphagov/govuk-frontend/blob/master/CHANGELOG.md

We're currently only on 3.9.1, so this needs to be updated across the project.